### ISSUE TITLE
Got sh: ALTERNATE_CONF: unknown operand even ALTERNATE_CONF already set

### ISSUE THREAD

**Issue by boshido** [WebHostingCoopTeam/rabbitmq-conf#1]
It may cause from shell syntax that ash can not use.
Anyway I'm not sure about this.
```
if [[ -v ALTERNATE_CONF ]]; then
```

Log
```
9/18/2017 4:27:00 PMsh: ALTERNATE_CONF: unknown operand
9/18/2017 4:27:00 PM2017-09-18T09:27:00Z rabbitmq-3-rabbitmq-1 /confd[1]: INFO Backend set to rancher
9/18/2017 4:27:00 PM2017-09-18T09:27:00Z rabbitmq-3-rabbitmq-1 /confd[1]: INFO Starting confd
9/18/2017 4:27:00 PM2017-09-18T09:27:00Z rabbitmq-3-rabbitmq-1 /confd[1]: INFO Backend nodes set to
9/18/2017 4:27:00 PM2017-09-18T09:27:00Z rabbitmq-3-rabbitmq-1 /confd[1]: INFO Using Rancher Metadata URL: http://rancher-metadata
9/18/2017 4:27:00 PM2017-09-18T09:27:00Z rabbitmq-3-rabbitmq-1 /confd[1]: INFO /etc/rabbitmq/rabbitmq.config has md5sum 22bf8582cfab6125c930f1e9a426423d should be d23270aab7f0c48124f101cfb7234567
9/18/2017 4:27:00 PM2017-09-18T09:27:00Z rabbitmq-3-rabbitmq-1 /confd[1]: INFO Target config /etc/rabbitmq/rabbitmq.config out of sync
9/18/2017 4:27:00 PM2017-09-18T09:27:00Z rabbitmq-3-rabbitmq-1 /confd[1]: INFO Target config /etc/rabbitmq/rabbitmq.config has been updated
9/18/2017 4:28:10 PM2017-09-18T09:28:10Z rabbitmq-3-rabbitmq-1 /confd[1]: INFO /etc/rabbitmq/rabbitmq.config has md5sum d23270aab7f0c48124f101cfb7234567 should be 22bf8582cfab6125c930f1e9a426423d
9/18/2017 4:28:10 PM2017-09-18T09:28:10Z rabbitmq-3-rabbitmq-1 /confd[1]: INFO Target config /etc/rabbitmq/rabbitmq.config out of sync
9/18/2017 4:28:10 PM2017-09-18T09:28:10Z rabbitmq-3-rabbitmq-1 /confd[1]: INFO Target config /etc/rabbitmq/rabbitmq.config has been updated
```

**Comment by case7026** [comment]
Still open.  Ran into this issue today when trying to set up a custom rabbitmq.config file using the Rancher catalog items to deploy a 3 node RabbitMQ cluster.

**Comment by joshuacox** [comment]
gladly accepting Pull requests on this one!

**Comment by case7026** [comment]
So it looks like this is because the busybox image this is built from does not include bash and only includes sh.

I'm going to submit a PR soon and switch the image to alpine:latest and add apk and bash to the Dockerfile so the conditional functions properly.

**Comment by joshuacox** [comment]
This appears to have been fixed with #4 @boshido can you test on your end?  I know this is a long time out, so I'm closing for now.  Feel free to re-open and add more details.
